src/lib/distances.jl

```diff
@@ -1,22 +1,23 @@
 using .Distances
+import .ChainRules: NO_FIELDS, rrule
```

Comment:
IIRC `NO_FIELDS` was deprecated a long time ago. Instead one should use just `NoTangent`.